compositor_render/src/wgpu/ctx.rs

```diff
@@ -76,7 +77,24 @@ impl WgpuCtx {
             format,
             utils,
             shader_parameters_bind_group_layout,
-            plane_cache,
+            plane,
+            empty_texture,
+        })
+    }
+
+    fn parameters_bind_group_layout(device: &wgpu::Device) -> wgpu::BindGroupLayout {
```

Comment:
it does not look like good place for this function. Probably correct module, but making it asociated function to WgpuCtx is confusing